test_get_count.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python3
+""" Test .get() and .count() methods
+"""
+from models import storage
+from models.state import State
+
+print("All objects: {}".format(storage.count()))
+print("State objects: {}".format(storage.count(State)))
+
+first_state_id = list(storage.all(State).values())[0].id
+print(first_state_id, '======')
+print("First state: {}".format(storage.get(State, first_state_id)))
```

Comment:
All methods are used properly and the style of the code is also good but it would be better if more comment is added to explain the code in simple terms.